skimage/transform/histogram_matching.py

```diff
@@ -19,12 +19,17 @@ def _match_cumulative_cdf(source, template):
     return interp_a_values[src_unique_indices].reshape(source.shape)
 
 
-
 def match_histograms(image, reference, multichannel=False):
     """Adjust an image so that its cumulative histogram matches that of another.
 
     The adjustment is applied separately for each channel.
 
+    For color image assume to use another colour space then RGB since the colour
```

Comment:
Hey, @Borda, thank you for helping us clarify the documentation; that's always valuable.

I am struggling to understand the new text.  Why do you say HSV is "continuous", and RGB is not?  I can help wordsmith this a bit, but I don't have a good understanding of the issue at hand.